### DISCUSSION TITLE
Sitemap.xml didn't work with this template.

### DISCUSSION THREAD

**Discussion by thinh-vu** [maximevaillancourt/digital-garden-jekyll-template#164] [Q&A]
Would someone kindly let me know of a way to generate a sitemap for a website that utilizes this template? This would improve the website's accessibility for Google Search Console to index its content. Currently, attempting to access the default sitemap.xml for the website results in a 404 error, such as: https://digital-garden-jekyll-template.netlify.app/sitemap.xml.

**Comment by maximevaillancourt** [comment]
Two potential solutions:
- Using [the official `jekyll/jekyll-sitemap` gem](https://github.com/jekyll/jekyll-sitemap) as a starting point and configuring as needed
- Creating the sitemap blueprint yourself (see [this blog post](https://www.independent-software.com/generating-a-sitemap-xml-with-jekyll-without-a-plugin.html))

**Reply by thinh-vu** [reply]
Thanks @maximevaillancourt for the update. I will try it.
